src/api/PostTalk.ts

```diff
@@ -2,25 +2,43 @@ import axiosInstance from '@redhat-cloud-services/frontend-components-utilities/
 
 import { Metadata } from '../types/Metadata';
 
+//{"options":[{"input":{"intents":null,"suggestion_id":null,"text":"cat"},"label":"cat"},{"input":{"intents":null,"suggestion_id":null,"text":"dog"},"label":"dog"},{"input":{"intents":null,"suggestion_id":null,"text":"kitten"},"label":"kitten"},{"input":{"intents":null,"suggestion_id":null,"text":"puppy"},"label":"puppy"}],"text":null}],"session_id":"f542f051-d303-4097-b49e-5b95e7e55adb"}
+
+// FIXME change options to fit in with the format
+export interface PostTalkOption {
+  label: string;
+  text: string;
+  input: {
+    intents: string | null;
+    suggestion_id: string | null;
+    text: string;
+  };
+}
+
 export interface PostTalkResponse {
-  recipient_id: number;
   text: string;
-  buttons?: Array<{
-    title: string;
-    payload: string;
-  }>;
-  custom?: CustomResponse;
+  options?: Array<PostTalkOption>;
+  command?: Command;
 }
 
-export interface CustomResponse {
+export interface Command {
+  args: string[];
   type: string;
   command?: string;
-  params?: object;
+  params?: string[];
+}
+
+export interface PostTalkResponseAPI {
+  response: PostTalkResponse[];
+  session_id: string;
 }
 
+// FIXME
 export const postTalk = async (message: string, metadata: Metadata) => {
-  return axiosInstance.post<unknown, Array<PostTalkResponse>>('/api/virtual-assistant/v1/talk', {
-    message,
+  return axiosInstance.post<unknown, PostTalkResponseAPI>('/api/virtual-assistant/v2/talk', {
+    input: {
+      text: message,
+    },
```

Comment:
Heads up, we are updating the returned types: 

https://github.com/RedHatInsights/astro-virtual-assistant-v2/pull/21/files#diff-2996752441ba0fd234f744ad35c2cbe970b62c1c5847938394be10582015cad9R103